### PR TITLE
Second Renault Video needs to have a different poster image

### PR DESCRIPTION
**Pull request by lps88**
## What does this change?

The latest Renault "Hosted By" video has a different poster image than the first one

Also changed the opengraph properties to use static image file instead of hardcoded s3 link, but need to test it works.
## Request for comment

@Calanthe @kelvin-chappell

### CODE REVIEW

File: commercial/app/views/hosted/episode1Video.scala.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
   preload="auto"
   style="width: 100%;"
   data-block-video-ads="true"
-  poster="@Static("images/commercial/renault-video-poster.jpg")"
+  poster="@Static("images/commercial/renault-video-poster-ep1.png")"
```
Comment (kelvin-chappell) [inline commercial/app/views/hosted/episode1Video.scala.html]: Could be `@page.poster` if page is passed through as parameter

Comment (kelvin-chappell) [inline commercial/app/views/hosted/episode1Video.scala.html]: Same for `teaserVideo.scala.html`

**Comment by kelvin-chappell** [comment]
Apart from repeating URL, looks good 👍

**Comment by paperboyo** [comment]
Whoa, stop the press! This image is 2MB 😆 Here it is a bit nicer (let me know if that's unacceptable) and 10 times smaller (images should go through the resizer, but these cannot, right?).

Also, we have removed all poster frames and rely on trailpics overlaid on videojs instead. Again, not sure if that would apply here...

renault-video-poster-ep1

**Comment by lps88** [comment]
@paperboyo thanks! can't believe I didn't notice the file size, oops. We're looking to use video atoms soon, and these pages are static prototypes for now so we'll have a better solution for the video/posters before long (hopefully) 😸

**Pull request merged**